web/.env

```diff
@@ -1,3 +1,6 @@
  POINT_URL=https://points-staging.hitcon.org/api/v1
+ ONLINE_URL=https://online-staging.hitcon.org
+ KOF_URL=https://kofserver.com
+ BOT_URL=https://t.me/hitcon_bot?start=00000001_6ce709f26bf3d745565024957ea1d003
```

Comment:
TODO: Use SPEC 3.10 /v1/telegram/generate-code to get a code and replace the fake code.